app/src/examples/index.ts

```diff
@@ -110,6 +110,7 @@ import WobbleExample from './WobbleExample';
 import WorkletExample from './WorkletExample';
 import WorkletRuntimeExample from './WorkletRuntimeExample';
 import NestedLayoutAnimationConfig from './LayoutAnimations/NestedLayoutAnimationConfig';
+// import ScreenAnimationExample from './ScreenAnimationExample';
```

Comment:
Let's uncomment this example
```suggestion
import ScreenAnimationExample from './ScreenAnimationExample';
```

Reply:
If you don't mind, I'll uncomment it after the release of `react-native-screens` because with the current latest version, the screen transition is not available. After the release, I will update the screens version in our example apps and uncomment this view.

Reply:
I don't want to uses version from github in package.json because it will break our TypeScript CI checks :/